BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceViewController.swift

```diff
@@ -173,45 +183,56 @@ final class DigitalInvoiceViewController: UIViewController {
         }
     }
 
-    override func viewDidLayoutSubviews() {
-        super.viewDidLayoutSubviews()
-        if UIDevice.current.isIphone {
-            if view.currentInterfaceOrientation.isLandscape {
-                if tableView.tableFooterView == nil {
-                    NSLayoutConstraint.deactivate(proceedViewConstraints)
-                    proceedView.removeFromSuperview()
-
-                    // frame is mandatory for tableview footer view, since it doesn't use autolayout, which means we have to specify size
-                    let container = UIView(frame: CGRect(x: 0,
-                                                         y: 0,
-                                                         width: view.bounds.width,
-                                                         height: Constants.buttonContainerHeight + Constants.padding))
-                    container.addSubview(proceedView)
-                    NSLayoutConstraint.activate([
-                        proceedView.leadingAnchor.constraint(equalTo: container.leadingAnchor),
-                        proceedView.trailingAnchor.constraint(equalTo: container.trailingAnchor),
-                        proceedView.bottomAnchor.constraint(equalTo: container.bottomAnchor),
-                        proceedView.topAnchor.constraint(equalTo: container.topAnchor, constant: Constants.padding)
-                    ])
-                    tableView.tableFooterView = container
-
-                    NSLayoutConstraint.activate(proceedViewTableConstraints)
-                    proceedView.isHidden = false
-                    bottomNavigationBar?.isHidden = true
-                }
-            } else {
-                if tableView.tableFooterView != nil {
-                    NSLayoutConstraint.deactivate(proceedViewTableConstraints)
-                    proceedView.removeFromSuperview()
-                    tableView.tableFooterView = nil
-                    proceedViewTableConstraints.last?.isActive = false
-                    view.addSubview(proceedView)
-                    NSLayoutConstraint.activate(proceedViewConstraints)
-                    proceedView.isHidden = configuration.bottomNavigationBarEnabled
-                    bottomNavigationBar?.isHidden = !configuration.bottomNavigationBarEnabled
-                }
-            }
+    override func viewWillTransition(to size: CGSize, with coordinator: any UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        updateFooterViewForIphoneOrientation()
+    }
+
+    private func updateFooterViewForIphoneOrientation() {
+        guard UIDevice.current.isIphone else { return }
+
+        let isLandscape = UIDevice.current.isLandscape
+
+        if isLandscape {
+            updateFooterForLandscape()
+        } else {
+            updateFooterForPortrait()
+        }
+    }
+
+    private func updateFooterForLandscape() {
```

Comment:
```suggestion
    private func configureFooterForLandscape() {
```
maybe even `configureNavBarForLandscape`